packages/kit/src/runtime/client/client.js

```diff
@@ -10,6 +16,7 @@ import { HttpError, Redirect } from '../control.js';
 import { stores } from './singletons.js';
 import { DATA_SUFFIX } from '../../constants.js';
 import { unwrap_promises } from '../../utils/promises.js';
+import * as devalue from 'devalue';
```

Comment:
will tree shaking remove the other stuff correctly if we write the import like this?

Reply:
Yep, Rollup doesn't care either way